### PR TITLE
Translations update from Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Weblate](https://translate.fedoraproject.org/projects/candlepin/master/) for candlepin/master.

Current translation status:

Weblate translation status

### CODE REVIEW

**Review by sonalidhome — APPROVED** [review]

**Review by sonalidhome — CHANGES_REQUESTED** [review]
This branch has conflicts.

**Comment by nikosmoum** [comment]
@sonalidhome Hi, conflicts can be resolved as descibred here: https://docs.weblate.org/en/weblate-4.5.1/faq.html#how-to-fix-merge-conflicts-in-translations
In this case the last commit in master (https://github.com/candlepin/candlepin/commit/313b6d430766d136e11aa6b8b6768bbc6dd5faec) removed a translation string, so we'd remove it entirely from this change in ko.po while resolving the conflict

**Comment by ptoscano** [comment]
This is because there are two entities (candlepin-bot and Weblate) doing changes to translations, and both of them assume they are the only ones doing changes. Weblate is actually able to update the existing translations with an updated message catalog template (.pot); please see my notes in a recent subscription-manager change related to this: https://github.com/candlepin/subscription-manager/pull/2485#issuecomment-799346450

In short:
1) drop the manual msgmerge of translations
2) enable the msgmerge addon in Weblate
3) profit

**Comment by nikosmoum** [comment]
@ptoscano Candlepin is already using the msgmerge addon in weblate, but this only updating the .po files based on what .pot has. 

The changes that candlepin-bot is doing is running gettext (to extract new translatable strings from source code into the .pot), and msgattrib --set-obsolete; msgattrib --no-obsolete (to remove obsolete strings from the .pot). There is no weblate addon for this work afaik, so we are doing it automated periodically (twice a week).

**Comment by ptoscano** [comment]
> running gettext (to extract new translatable strings from source code into the .pot),

This is fine.

> and msgattrib --set-obsolete; msgattrib --no-obsolete (to remove obsolete strings from the .pot)

ITYM from the .po files. I think this is a bad idea: an obsolete translation can be used again in case a message reappears in the pot file, with the fuzzy flag it had before; also, obsolete translations are never included in compiled .mo files.

What is the reason for doing this step? I'd ask for this to be reconsidered and dropped.

**Comment by nikosmoum** [comment]
@ptoscano @sonalidhome Alright, as discussed, we'll stop the automated candlepin-bot removing obsolete strings from the .po files and allow fuzzy matching (I'll add a card to the backlog for this), but for the time being, we need to resolve this conflict
